docs/Tutorials/Contracts/Deploying-Contracts.md

```diff
@@ -4,11 +4,11 @@ description: deploying smart contracts
 
 # Deploying smart contracts to an Ethereum chain
 
-Use the [Developer Quickstart](../Developer-Quickstart.md) to rapidly generate local blockchain
-networks.
-
 This tutorial shows you how to deploy smart contracts as transactions to a network.
 
+You can use the [Developer Quickstart](../Developer-Quickstart.md) to rapidly generate local blockchain
+networks.
+
 ## Prerequisites
 
 * A [private network](../Examples/Private-Network-Example.md) if deploying a public contract.
```

Comment:
realise you didn't change this - but the "private network" vs "privacy-enabled network" is a bit confusing. have we tried to come up with something better?

actually maybe the "you can use the quickstart" from line 9 would fit in here - this tutorial uses a local blockchain, you can use the quickstart to rapidly generate one... and if deploying a private contract, you need to enable privacy